src/AppInstallerCommonCore/Runtime.cpp

```diff
@@ -309,6 +312,36 @@ namespace AppInstaller::Runtime
                 result = GetKnownFolderPath(FOLDERID_Profile);
                 create = false;
                 break;
+            case PathName::PortableAppUserRoot:
+                result = Settings::User().Get<Setting::PortableAppUserRoot>();
+                if (result.empty())
+                {
+                    result /= GetKnownFolderPath(FOLDERID_LocalAppData);
+                    result /= s_SecureSettings_Base;

```

Comment:
>s_SecureSettings_Base [](http://example.com/codeflow?start=30&length=21)

This should be another const with same value but not s_SecureSettings_Base

Reply:
created separate consts specific to portables